### PR TITLE
[ElasticDL] Add ElasticDL integration test and k8s setup on Travis

### PR DESCRIPTION
**Pull request by terrytangyuan**
Part of #664.

Note that the test is not enabled as ElasticDL is not open sourced yet (left a TODO in `.travis.yml`). 

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,6 +41,15 @@ script:
...
   - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:latest bash scripts/test_e2e.sh
+  # Install kubectl and minikube (currently only used for ElasticDL integration tests with maxcompute)
+  # TODO(terrytangyuan): Enable TestEnd2EndMaxComputeElasticDL test in scripts/test_maxcompute.sh once ElasticDL is open sourced
+  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v$K8S_VERSION/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v$MINIKUBE_VERSION/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+  - mkdir -p $HOME/.kube $HOME/.minikube
+  - touch $KUBECONFIG
+  - sudo minikube start --vm-driver=none --kubernetes-version=v$K8S_VERSION --cpus 2 --memory 6144
+  - "sudo chown -R travis: /home/travis/.minikube/"
+  - kubectl cluster-info
```
Comment (tonyyang-svail) [inline .travis.yml]: Maybe combine the following command to a bash script?

Comment (terrytangyuan) [reply .travis.yml]: Good idea. Done.

File: .travis.yml [hunk 2]
```diff
@@ -49,6 +50,7 @@ deploy:
...
+    - secure: f03CHRo2Afl9PD6jLuKtY/6b/MgJkytcfeIhSrAYZFLzUCQTa4MI5yih4pG/VJtTpKVpoQD7NJikj/x0eO/6NOGxegJwaruY8EDg/4LVLeuVqRipmw0V9l3LcZ7AofxPAeJ6RZLpHALR9cYegXEEtvyfjNRw2BhQZsZBvTE8sbbvfzLJOY1UC9CeQnHxDGZYXG3WSMr7b+44ifg4OBweATmEsdIVPD7kgexunJvHw0vzQ25vsf+huYwLorVc7uRghoYY8SoaLMEcmP50Jsl2Mn7AUoFUyM3tl/kOub11BV4/yGrGJC1BysT7l48frSKYqSU1yeT+0xiJjgkOwpV8BcxOmXrgm2s6PDzWDmiXtBRhE6n79iG2IuuTuYLk4n/Rm7uwKYu6EdnooKSnhGV6DW7mLVJ5Rcb655xCB+AoXb79Ilp60YxsMFTqzdhMkf3QpQV/OsEHFrzNhab6+pWcrwOeCq/CXU0MDO0rQuX4YIBwdEyAlDrj2JDvJ868lpdTJTxUUgUovH2j8rFrtCf5Bx0gqg0Cj81s/7cMdhtoV+wbTtHoQ7TLTZkvqYyFAMCkl2d5g00UhZpq+HTjzcPOL5xKc0mZhsuYq03nTVMQWzIJiKkaWbDPoY3eZcLga0cO8VD2HgzsQ5lg300ZTF73AxonZheyzJVWtLQvrpT8dPI=
```
Comment (tonyyang-svail) [inline .travis.yml]: maybe leave this line unchanged?

**Review by tonyyang-svail — COMMENTED** [review]
@terrytangyuan Shall we put this PR on hold until EDL is publicly available?

**Comment by terrytangyuan** [comment]
@tonyyang-svail I think it's probably fine to get this in first so I can test `TestEnd2EndMaxComputeElasticDL` together with other changes more easily locally.

**Review by tonyyang-svail — APPROVED** [review]

**Pull request merged**